scripts/generate-google3-docs/resource-reference/templates/apikeys_apikeyskey.tmpl

```diff
@@ -0,0 +1,58 @@
+{{template "headercomment.tmpl" .}}
```

Comment:
Mhmm, maybe remove the resource doc?
We didn't add any resource doc for any autogen / v1alpha1 resources because we believe additional information about no resource quality guaranteed, no backwards compatibility support, and etc should be provided on the page, but we never get the chance to prioritize working with PM to finalize the idea.

Reply:
Removed the alpha docs.  We can add them back in a separate PR, we should also make sure they are readable on github for those alpha resources - a good opportunity.